src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -21,7 +21,7 @@ import {
   JOB_STATUSES,
   TRANSLATION_SUFFIX,
   EDITING_STEPS,
-  FILTERS,
+  FILTERS, ERRORS,
```

Comment:
plz move to the next line - common codestyle

Reply:
moved!